### PR TITLE
fix: do not create case when stmts for comparison operators

### PR DESCRIPTION
**Pull request by johannes-vogel**

### CODE REVIEW

**Review by BobdenOs — APPROVED** [review]

**Review by danjoa — COMMENTED** [review]
Let's pls review in sync meeting

File: hana/lib/HANAService.js [hunk 1]
```diff
@@ -646,21 +646,22 @@ class HANAService extends SQLService {
...
-        '<=': null,
-        '!<': null,
-        '!>': null,
+        // '>': null,
+        // '<': null,
+        // '<>': null,
+        // '>=': null,
+        // '<=': null,
+        // '!<': null,
+        // '!>': null,
```
Comment (danjoa) [inline hana/lib/HANAService.js:659]: Would that change specified behavior?

Comment (BobdenOs) [reply hana/lib/HANAService.js:659]: This change only impacts the usage of these operators where `HANA` does not allow for them.

```SQL
SELECT 1>0 as gt FROM DUMMY
```

Currently considering to close this PR as https://github.com/cap-js/cds-dbs/pull/426 does the same with more fixes.

**Comment by johannes-vogel** [comment]
obsolete due to #426

**Pull request closed without merging**